pkgs/development/python-modules/pillow-heif/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, cmake
+, pillow
+, pytest
+, nasm
+, libheif
+, libaom
+, libde265
+, x265
+}:
+
+buildPythonPackage rec {
+  pname = "pillow_heif";
+  version = "0.13.0";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "bigcat88";
+    repo = "pillow_heif";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-GbOW29rGpLMS7AfShuO6UCzcspdHtFS7hyNKori0otI=";
+  };
+
+  nativeBuildInputs = [ cmake nasm ];
+  propagatedBuildInputs = [ libheif libaom libde265 pillow x265 ];
```

Comment:
Do the C libs have to be explicitly propagated? Otherwise please move them to `buildInputs`. If they're referenced in shared objects then it should be fine.